plugins/utils/commands/address-reviews.md

```diff
@@ -0,0 +1,185 @@
+---
+description: Fetch and address all PR review comments
+argument-hint: [PR number (optional - uses current branch if omitted)]
+---
+
+## Name
+utils:address-reviews
+
+## Synopsis
+/utils:address-reviews [PR number (optional - uses current branch if omitted)]
+
+## Step 0: Checkout the PR Branch
```

Comment:
should this steps be ### below ##implementation according to ref impl https://github.com/openshift-eng/ai-helpers/blob/main/plugins/hello-world/commands/echo.md